bin/_lab

```diff
@@ -46,6 +46,5 @@ main();
 
 process.on('exit', async () => {
 
-    const rimraf = Util.promisify(Rimraf);
-    await rimraf('./test_runner');
+    await Fs.rmdir('./test_runner', { recursive: true });
```

Comment:
Considering `rimraf` is the equivalent of `rm -rf`, shouldn't we use `Fs.rm('./test_runner', { recursive: true, force: true });`?

Per Node docs: https://nodejs.org/docs/latest-v16.x/api/fs.html#fspromisesrmdirpath-options
> To get a behavior similar to the rm -rf Unix command, use [fsPromises.rm()](https://nodejs.org/docs/latest-v16.x/api/fs.html#fspromisesrmpath-options) with options { recursive: true, force: true }.

Reply:
Sure.